### DISCUSSION TITLE
[v1.7] discord.Profile fetching

### DISCUSSION THREAD

**Discussion by ModerNews** [Rapptz/discord.py#6945] [Q&A] [answered]
Just a quick question, since version 1.7 made `Client.fetch_user_profile()` deprecated is there any other (and working for bots, cause I know this was a problem with function I'm mentioning) way of fetching user's profile? I mean with all the information that only `discord.Profile` class is storing (like being a nitro owner, etc.)

**Comment by Rapptz** [comment, marked as answer]
That endpoint is only accessible for user bots. Real bots cannot fetch profile information and Discord does not expose a way.
